src/Plugin/ConceptSubset/DeprecatedCountries.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_corporate_countries\Plugin\ConceptSubset;
+
+use Drupal\Core\Entity\Query\QueryInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\rdf_entity\RdfFieldHandlerInterface;
+use Drupal\rdf_skos\ConceptSubsetPluginBase;
+use Drupal\rdf_skos\Plugin\PredicateMapperInterface;
+
+/**
+ * Creates a subset of the countries vocabulary.
+ *
+ * @ConceptSubset(
+ *   id = "deprecated_countries",
+ *   label = @Translation("Deprecated countries"),
+ *   description = @Translation("Filters out deprecated countries."),
+ *   predicate_mapping = TRUE,
+ *   concept_schemes = {
+ *     "http://publications.europa.eu/resource/authority/country"
+ *   }
+ * )
+ */
+class DeprecatedCountries extends ConceptSubsetPluginBase implements PredicateMapperInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alterQuery(QueryInterface $query, $match_operator, array $concept_schemes = [], string $match = NULL): void {
+    $query->condition('deprecated', 'false');
```

Comment:
I used a check in the country repository for true or 1. Maybe we should create a condition and put both false or 0, but I don't see any in the vocabulary. Let's go like it is now.